posthog/urls.py

```diff
@@ -53,11 +53,27 @@ def home(request, *args, **kwargs):
 def authorize_and_redirect(request):
     if not request.GET.get("redirect"):
         return HttpResponse("You need to pass a url to ?redirect=", status=401)
-    url = request.GET["redirect"]
+    if not request.META.get("HTTP_REFERER"):
+        return HttpResponse('You need to make a request that includes the "Referer" header.', status=400)
```

Comment:
Should this redirect only be to "authorized domains"?

I think referrer can be manually set on the request. So I make a request. Get the 400 then manually set the referrer to `evil.com` and still get redirected

But things like login and launching the toolbar should only be on authorized domains which you have to be logged in to set

Reply:
Oops, didn't see this before merging. Probably would have merged regardless to already improve on the current state. That's probably a good idea though.